app/hotels/src/map/allHotels/HotelSwipeList.js

```diff
@@ -8,10 +8,10 @@ import {
   BottomSheet,
   Device,
   StyleSheet,
-  type OnLayout,
   BottomSheetHandle,
   Translation,
 } from '@kiwicom/mobile-shared';
+import type { LayoutEvent } from 'react-native/Libraries/Types/CoreEventTypes';
```

Comment:
Would you rather have a layer from a common package instead of importing directly from `react-native`?

Reply:
Probably a good idea, it will be easier to remember.

Reply:
@RobinCsl 🚀